### ISSUE TITLE
Redner's Markets in index is not actually used

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/name-suggestion-index#2518]
Added in https://github.com/osmlab/name-suggestion-index/commit/6b17cd6fbd081c4e46ce634cc3e7760428e2cc25

> Redner's Markets, Inc. is an American supermarket chain that currently operates 44 Warehouse Markets and 20 Redner's Quick Shoppe convenience stores located in Pennsylvania, Delaware, and Maryland . Redner's Warehouse Markets has a distribution warehouse located in Maidencreek Township, Pennsylvania. 

from https://en.wikipedia.org/wiki/Redner's_Markets

In OSM it is mapped like this: https://www.openstreetmap.org/way/571730855 with `name=Redner's Warehouse Markets`

Not sure whatever name in index should be changed or match added.

Current entry 

```
  "shop/supermarket|Redner's Markets": {
    "nocount": true,
    "tags": {
      "brand": "Redner's Markets",
      "brand:wikidata": "Q7306166",
      "brand:wikipedia": "en:Redner's Markets",
      "name": "Redner's Markets",
      "shop": "supermarket"
    }
  },
```

**Comment by matkoniecz** [comment]
ping @maxerickson

**Comment by maxerickson** [comment]
I guess it is mostly mapped as "Redner's":

http://overpass-turbo.eu/s/Ho1

That's probably the correct name, as it appears to be the big name on the sign on the stores and how they brand their ads and so on.

**Comment by matkoniecz** [comment]
@maxerickson Are you interested in updating the entry to match this?

**Comment by maxerickson** [comment]
Addressed in 54ccc06.
